src/onemod/backend/jobmon_backend.py

```diff
@@ -257,17 +360,28 @@ def get_tasks(
     """
     if isinstance(model, Pipeline):
         return get_pipeline_tasks(
-            model, method, tool, resources, python, stages, **kwargs
+            model,
+            method,
+            tool,
+            resources,
+            python,
+            stages,
+            external_upstream_tasks,
+            task_and_template_prefix,
+            max_attempts,
+            **kwargs,
         )
     return get_stage_tasks(
```

Comment:
I think we also need to pass `upstream_tasks=external_upstream_tasks` to `get_stage_tasks()`